package.json

```diff
@@ -9,7 +9,7 @@
     "start": "sh scripts/start.sh",
     "dev": "sh scripts/dev.sh",
     "test": "sh scripts/test.sh",
-    "test:watch": "yarn jest -- --watch",
+    "test:watch": "yarn jest -- --watch --runInBand",
```

Comment:
@kanaabe - I think this needs to be on line 11? `test:watch` is just for development mode and should never be run in CI

Reply:
Yeahh I added it to both. See `scripts/test.sh`

Reply:
Ooh 👍